File: Python-programming-1/fibonacci.py
def fibonacci(n):
    a = 0
    b = 1
     
    if n < 0:
        return -1
    elif n == 0:
        return 0
    elif n == 1:
        return b
    else:
        for i in range(1, n):
            c = a + b
            a = b
            b = c
        
        return b

print(fibonacci(10))

File: Java Fundamentals programs/Fibonacci.java
class Fibonacci
{
    public static void main(String args[]) 
    {
        int num1=0, num2=1, num3, i, count=10 ;
        System.out.print(num1 + " " +num2); // i=0 and i=1 i.e. num1 and num2 will be printed 

        for(i=2; i<count; ++i)  //loop starts
        {
           num3 = num1 + num2;
           System.out.print(" " + num3);
           num1= num2;
           num2 = num3;
        }
    }
}
